healsparse/healSparseMap.py

```diff
@@ -1378,6 +1386,38 @@ def get_single(self, key, sentinel=None, copy=False):
         return HealSparseMap(cov_map=self._cov_map, sparse_map=new_sparse_map,
                              nside_sparse=self._nside_sparse, sentinel=_sentinel)
 
+    def astype(self, dtype, sentinel=None):
```

Comment:
I didn't think of this, it's a nice addition!

Reply:
This is in response to an ancient issue, #36 .  I don't want people mucking with private variables, but it's clear that it is useful to be able to convert types correctly!